tests/integration/targets/aci_fabric_node_decommission/tasks/main.yml

```diff
@@ -0,0 +1,260 @@
+# Test code for the ACI modules
+# Copyright: (c) 2017, Bruno Calogero <brunocalogero@hotmail.com>
+# Copyright: (c) 2023, Gaspard Micol <gmicol@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  ansible.builtin.fail:
+    msg: "Please define the following variables: aci_hostname, aci_username and aci_password."
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+- name: Set vars
+  ansible.builtin.set_fact:
+    aci_info: &aci_info
+      host: "{{ aci_hostname }}"
+      username: "{{ aci_username }}"
+      password: "{{ aci_password }}"
+      validate_certs: "{{ aci_validate_certs | default(false) }}"
+      use_ssl: "{{ aci_use_ssl | default(true) }}"
+      use_proxy: "{{ aci_use_proxy | default(true) }}"
+      output_level: '{{ aci_output_level | default("info") }}'
+
+# CLEAN ENVIRONMENT
+- name: Ensure that the fabric node exists
+  cisco.aci.aci_fabric_node:
+    <<: *aci_info
+    name: "{{ aci_fabric_node_name }}"
+    node_type: unspecified
+    pod_id: "{{ aci_fabric_pod_id }}"
+    node_id: "{{ aci_fabric_node_id }}"
+    serial: "{{ aci_fabric_serial }}"
+    state: present
+
+# SETUP
+- name: Commission the fabric node
+  cisco.aci.aci_fabric_node_decommission:
+    <<: *aci_info
+    pod_id: 1
+    node_id: 201
+    state: absent
+  register: commission_node
+
+- name: Query fabric node to check the node is commissioned - $query_commission_node_1
+  cisco.aci.aci_fabric_node_decommission:
+    <<: *aci_info
+    pod_id: 1
+    node_id: 201
+    state: query
+  register: query_commission_node_1
+
+- name: Assertion check for query fabric node to check the node is commissioned
+  ansible.builtin.assert:
+    that:
+      - query_commission_node_1 is not changed
+      - query_commission_node_1.current == []
+
+# TEST
+- name: Decommission the fabric node (check_mode) - $cm_decommission_node
```

Comment:
what is this $cm_decommission_node? is this new standard?

Reply:
Removed, added by mistake